src/test/ui/const-generics/const_evaluatable_checked/from-sig-fail.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0080]: evaluation of constant value failed
+  --> $DIR/from-sig-fail.rs:4:35
+   |
+LL | fn test<const N: usize>() -> [u8; N - 1] {
+   |                                   ^^^^^ attempt to compute `0_usize - 1_usize` which would overflow
```

Comment:
Can we somehow tell the user that this is speciically about `test<0>`?

Reply:
Yeah, we should. Will do so in another PR though.